form/2d/p5js/crisscross/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: p5js
+author: "Paulina Mrksic"
+author-url: "https://github.com/paulinalinaa"
+title: "Law of closure and contrast"
```

Comment:
This title is to generic. Please use something more specific e.g. "Criss Cross" 

![](https://media.giphy.com/media/ujJ7AU4jHUXiE/giphy.gif)